src/dpp/cluster/guild.cpp

```diff
@@ -150,4 +150,13 @@ void cluster::guild_sync_integration(snowflake guild_id, snowflake integration_i
 }
 
 
+void cluster::guild_get_onboarding(dpp::onboarding o, dpp::command_completion_event_t callback) {
```

Comment:
guild get onboarding should take guild id not onboarding as it's first parameter imho

Reply:
I thought it can take the guild id from the onboarding object. But yeah ill change it.